hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/service/launcher/IrqHandler.java

```diff
@@ -39,7 +38,7 @@
 @InterfaceAudience.Private
 @InterfaceStability.Unstable
 @SuppressWarnings("UseOfSunClasses")
```

Comment:
Can this warning suppression be removed now?

Reply:
removed